# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/clucene/hotfix-pthread.patch.musl
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- clucene-core-2.3.3.4/src/shared/CLucene/LuceneThreads.h.vanilla
+++ clucene-core-2.3.3.4/src/shared/CLucene/LuceneThreads.h
@@ -7,6 +7,7 @@
 #ifndef _LuceneThreads_h
 #define  _LuceneThreads_h
 
+#include <pthread.h>
 
 CL_NS_DEF(util)
 class CLuceneThreadIdCompare;

